graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -280,7 +287,32 @@ def _check_for_cascading_type_suppression(
         raise CascadingSuppressionError("\n".join(error_message_components))
 
 
-def _ensure_no_unsupported_suppression(
+def _ensure_no_unsupported_operations(
+    schema_ast: DocumentNode, renamings: Mapping[str, Optional[str]], scalars: AbstractSet[str],
+) -> None:
+    """Check for unsupported renaming or suppression operations."""
+    _ensure_no_unsupported_scalar_operations(renamings, scalars)
+    _ensure_no_unsupported_suppressions(schema_ast, renamings)
+
+
+def _ensure_no_unsupported_scalar_operations(
+    renamings: Mapping[str, Optional[str]], scalars: AbstractSet[str],
+) -> None:
+    """Check for unsupported scalar operations."""
+    unsupported_scalar_operations = {}  # Map scalars to value to be renamed.
+    for scalar in scalars:
+        if renamings.get(scalar, scalar) != scalar:
+            # Then it either attempts to rename or suppress the scalar.
```

Comment:
The intent here is that `renamings` attempts to do _something_ with the scalar, maybe renaming or maybe suppression. This is the case iff `renamings.get(scalar, scalar)` returns something that is _not_ `scalar`

Reply:
Yeah, I figured that's what's happening. I'd just rephrase the comment to clarify it, since it's unusual for a `Then` (especially a capitalized one at the start of a sentence) to be matched up with the `if` in the code. It definitely threw me off for a bit when I first read it.